eole/config/models.py

```diff
@@ -36,6 +36,11 @@ class EmbeddingsConfig(Config):
         description="Absolute number of positions to learn "
         "position embeddings on (position_encoding_type: Learned)",
     )
+    position_shift: int | None = Field(
+        default=0,
+        description="Positions IDS shift before making position embed "
+        "dirty patch to cover for xlm-roberta-xl",
+    )
```

Comment:
Shouldn't this be set in xlm-roberta-xl conversion if this is a necessary patch?

Reply:
yes I know I forgot because I added it directly in the config.json will be in for next commit